src/python/pants/backend/project_info/list_targets.py

```diff
@@ -27,20 +27,6 @@ def register_options(cls, register):
                 "--provides-columns."
             ),
         )
-        register(
-            "--provides-columns",
-            default="address,artifact_id",
-            help=(
-                "Display these columns when --provides is specified. Available columns are: "
-                "address, artifact_id, repo_name, repo_url, push_db_basedir"
-            ),
-            removal_version="2.0.1.dev0",
```

Comment:
This was a typo. I meant for it to be 2.1.0.dev0 I'm sure. Can we bump this version instead?

Reply:
Hrm. OK.